addons/pos_restaurant/static/src/app/control_buttons/control_buttons.js

```diff
@@ -51,22 +51,21 @@ patch(ControlButtons.prototype, {
         const orderUuid = this.pos.get_order().uuid;
         this.pos.get_order().setBooked(true);
         this.pos.showScreen("FloorScreen");
-        document.addEventListener(
-            "click",
-            async (ev) => {
-                if (this.pos.isOrderTransferMode) {
-                    this.pos.isOrderTransferMode = false;
-                    const tableElement = ev.target.closest(".table");
-                    if (!tableElement) {
-                        return;
-                    }
-                    const table = this.pos.getTableFromElement(tableElement);
-                    await this.pos.transferOrder(orderUuid, table);
-                    this.pos.setTableFromUi(table);
-                }
-            },
-            { once: true }
-        );
+        const onClickWhileTransfer = async (ev) => {
```

Comment:
is there a way to do `document.removeEventListener` with anonymous functions ?

Reply:
This is correct, because the previous declaration of the listener with `{ once: true }` doesn't work anymore. So we need declare the handler, and remove the handler when it is _not_ ignored.